banks.json

```
@@ -1,4 +1,39 @@
 {
+  "bunq": {
+    "bankName": "bunq",
```

```suggestion
    "bankName": "BUNQ",
```

This is the bank name as found in the IBAN. So this should be all caps